addons/stock/report/report_deliveryslip.xml

```diff
@@ -46,12 +46,12 @@
                         <span t-field="o.name">WH/OUT/0001</span>
                     </h2>
                     <div class="oe_structure"></div>
-                    <div class="row mt32 mb32">
-                        <div t-if="o.origin" class="col-auto col-3 mw-100 mb-2" name="div_origin">
+                    <div class="mt32 mb32" style="overflow: auto;">
+                        <div t-if="o.origin" class="float-4 mw-100 mb-2" name="div_origin">
```

Comment:
instead of float-4 that's a little weird (currently there is the bootstrap naming "col-3" means 25% in relation to col-12 that is 100%, or w-25 meaning width: 25%), maybe you could just use existing classes:

```suggestion
                        <div t-if="o.origin" class="float-start w-25 mb-2" name="div_origin">
```